commands/user.go

```diff
@@ -315,3 +329,45 @@ auth_service: {{.AuthService}}`
 
 	return nil
 }
+
+func listUsersCmdF(c client.Client, command *cobra.Command, args []string) error {
+	// TODO What to do with err?
+	page, errp := command.Flags().GetInt("page")
+	if errp != nil {
+		return errors.Wrap(errp, "Could not parse --page")
+	}
+	perPage, errpp := command.Flags().GetInt("per-page")
+	if errpp != nil {
+		return errors.Wrap(errp, "Could not parse --per-page")
+	}
+	showAll, errs := command.Flags().GetBool("all")
+	if errs != nil {
+		return errors.Wrap(errs, "Could not parse --all")
+	}
+
+	if showAll {
+		page = 0
+	}
+
+	tpl := `{{.Id}}: {{.Username}} ({{.Email}})`
+
+	users, response := c.GetUsers(page, perPage, "")
+	if response.Error != nil {
+		return errors.Wrap(response.Error, "Failed to fetch users")
+	}
+	for ; users != nil && len(users) > 0; users, response = c.GetUsers(page, perPage, "") {
+		if response.Error != nil {
+			return errors.Wrap(response.Error, "Failed to fetch users")
+		}
+		for _, user := range users {
+			printer.PrintT(tpl, user)
+		}
+
+		if !showAll {
```

Comment:
I don't think this is needed anymore now the `page = 0` gets set when showAll is true. This will not print any more pages in the normal mode (when page and perPage is passed)

Reply:
If we ignore this condition, then all users will be printed starting from page `page` even if `showAll` is `false`.

Reply:
I thought that was the intention ? Otherwise we should not mention "Start page", because that means the start offset from where all pages would be printed.

@mgdelacroix - Can you clarify ?

Reply:
The intent was for the command to either show all the users when the `--all` flag is true, or one page of the user list. I agree with ditching "start page" in favor of something clearer

Reply:
Maybe something like "Page number to fetch for the list of users" ?